src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,110 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertData = "INSERT INTO manufacturers(name,country) values(?,?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                        PreparedStatement createManufacturerStatement =
+                                connection.prepareStatement(insertData,
+                                        Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
```suggestion
        try (Connection connection = ConnectionUtil.getConnection();
                PreparedStatement createManufacturerStatement =
                        connection.prepareStatement(insertData, Statement.RETURN_GENERATED_KEYS)) {
```
do we realy need so many spaces?

Reply:
Same thing in other try with recources

Reply:
maven ругается из-за длины строки

Reply:
We need use only 8 spaces after line wraping. Y can fill free to wraping lines but you neet fix intendation